src/v3/components/time_picek/index.tsx

```diff
@@ -0,0 +1,164 @@
+import { DesktopTimePicker, LocalizationProvider, renderTimeViewClock } from '@mui/x-date-pickers';
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
+import { Box, ClickAwayListener, Stack, TextFieldProps, useMediaQuery } from '@mui/material';
+import { Button, TextField } from '@components';
+import { Dispatch, SetStateAction, useEffect, useState } from 'react';
+import { CPETimePickerProps } from './time_picker.types';
+import { StyledIconWrapper } from '../user_card/user_card.styles';
```

Comment:
@eloyEntony: would it be possible to update this one to not use other styles in another component. If some styles in another component become useful for other, you can propose them to be moved in a shared location and file, and all other components will get it from there. Thanks.